src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -176,17 +177,6 @@ const ArticlePage = ({ pageData, mostReadEndpointOverride }) => {
     filterForBlockType(promoImageBlocks, 'rawImage'),
   );
 
-  const MostReadWrapper = ({ children }) => (
-    <MostReadSection css={styles.mostReadSection}>
-      <MostReadSectionLabel mobileDivider={showRelatedTopics && topics} />
```

Comment:
We'll need to pass in the `mobileDivider` to the new MostRead component based on this same logic